From 497efe8066b12db8df722ce084b7113fe5fa8c64 Mon Sep 17 00:00:00 2001
From: Dong Aisheng <b29396@freescale.com>
Date: Wed, 6 May 2015 19:23:01 +0800
Subject: [PATCH] MLK-10854 dts: imx6ul: add missed dtbs into dts/Makefile

Those dtbs are missed in dts/Makefile.
Then 'make ARCH=arm dtbs' command will miss to compile them.

Signed-off-by: Dong Aisheng <b29396@freescale.com>
---
 arch/arm/boot/dts/Makefile | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index 70e2b5e..63b011c 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -210,6 +210,10 @@ dtb-$(CONFIG_ARCH_MXC) += \
 	imx6sx-sdb-lcdif1.dtb \
 	imx6ul-ddr3-arm2.dtb \
 	imx6ul-ddr3-arm2-flexcan2.dtb \
+	imx6ul-ddr3-arm2-lcdif.dtb \
+	imx6ul-ddr3-arm2-mqs.dtb \
+	imx6ul-ddr3-arm2-spdif.dtb \
+	imx6ul-ddr3-arm2-wm8958.dtb \
 	imx6ul-evk.dtb \
 	vf610-cosmic.dtb \
 	imx6sx-sdb-emmc.dtb \
-- 
1.8.0

